builder.toml.tpl

```diff
@@ -2,11 +2,11 @@
 buildpacks = [
   { id = "io.projectriff.command",       uri = "https://storage.googleapis.com/projectriff/command-function-buildpack/io.projectriff.command-0.2.0-BUILD-SNAPSHOT-20200304161811-a66028adfc6d2ab6.tgz" },
   { id = "io.projectriff.java",          uri = "https://storage.googleapis.com/projectriff/java-function-buildpack/io.projectriff.java-0.3.0-BUILD-SNAPSHOT-20200324133014-3be580e2930f3290.tgz" },
-  { id = "io.projectriff.node",          uri = "https://storage.googleapis.com/projectriff/node-function-buildpack/io.projectriff.node-0.3.0-BUILD-SNAPSHOT-20200324133839-82d667c353f35709.tgz" },
+  { id = "io.projectriff.node",          uri = "https://storage.googleapis.com/projectriff/node-function-buildpack/io.projectriff.node-0.3.0.tgz" },
 
   { id = "org.cloudfoundry.buildsystem", uri = "https://storage.googleapis.com/cnb-buildpacks/build-system-cnb/org.cloudfoundry.buildsystem-{{ go mod download -json | jq -r 'select(.Path == "github.com/cloudfoundry/build-system-cnb").Version' | sed -e 's/^v//g' }}.tgz" },
   { id = "org.cloudfoundry.node-engine", uri = "https://github.com/cloudfoundry/node-engine-cnb/releases/download/{{ go mod download -json | jq -r 'select(.Path == "github.com/cloudfoundry/node-engine-cnb").Version' }}/node-engine-cnb-{{ go mod download -json | jq -r 'select(.Path == "github.com/cloudfoundry/node-engine-cnb").Version' | sed -e 's/^v//g' }}.tgz" },
-  { id = "org.cloudfoundry.yarn-install",        uri = "https://github.com/cloudfoundry/yarn-install-cnb/releases/download/v0.1.14/yarn-install-cnb-0.1.14.tgz" },
+  { id = "org.cloudfoundry.yarn-install",        uri = "https://github.com/cloudfoundry/yarn-install-cnb/releases/download/v0.1.16/yarn-install-cnb-0.1.16.tgz" },
```

Comment:
how about we template this like the other cf buildpacks

Reply:
ah, it was only templated in the `.tpl.tpl`, not in the `.tpl`

Reply:
Where is the matryoshka emoji when we need it? :(